.openpublishing.redirection.json

```diff
@@ -242,6 +242,26 @@
     {
         "source_path": "docs/tutorials/index.md", 
         "redirect_url": "/dotnet/articles/samples-and-tutorials/"
+    },
+    {
+        "source_path": "docs/csharp/getting-started/consuming-library-with-visual-studio-2017.md", 
+        "redirect_url": "/dotnet/articles/getting-started/consuming-library-with-visual-studio",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs/csharp/getting-started/debugging-with-visual-studio-2017.md", 
```

Comment:
Double-301? ... If a proxy or a system has a 301 on the non-2017 address and they go to the 2017-version of the doc ... it 301's back to the non-2017 doc. Hum. 🤔